vpr/src/place/noc_place_utils.cpp

```diff
@@ -163,12 +229,26 @@ void re_route_associated_traffic_flows(ClusterBlockId moved_block_router_id, Noc
         for (auto& traffic_flow_id : *assoc_traffic_flows) {
             // first check to see whether we have already re-routed the current traffic flow and only re-route it if we haven't already.
             if (updated_traffic_flows.find(traffic_flow_id) == updated_traffic_flows.end()) {
+                // get all links for this flow route before it is rerouted
+                // The returned const std::vector<NocLinkId>& is copied so that we can modify (sort) it
```

Comment:
Not sure why it needs to be sorted; if this is some important invariant it would be good to comment it.